user_tools/src/spark_rapids_pytools/common/utilities.py

```diff
@@ -117,6 +117,14 @@ def get_rapids_tools_env(cls, k: str, def_val=None):
     def set_rapids_tools_env(cls, k: str, val):
         os.environ[cls.find_full_rapids_tools_env_key(k)] = str(val)
 
+    @classmethod
+    def get_or_set_rapids_tools_env(cls, k: str, default_val) -> str:
+        current_val = cls.get_rapids_tools_env(k)
+        if current_val is None or (isinstance(current_val, str) and current_val == ''):
+            cls.set_rapids_tools_env(k, default_val)
+            return str(default_val)
+        return current_val
+
```

Comment:
I wonder if it is feasible to add extra default argument to `get_rapids_tools_env` instead of keeping 2 methods that almost do the same thing? I am fine if you want to rename the existing method to `get_or_set_*`